File: python/ex051.py
# Faça um aplicativo que leia o preço de 8 produtos. No final, mostre na tela
# qual foi o maior e qual foi o menor preço digitados.

counter = 1
majorProduct = 0
minorProduct = 999
inputPrice = 0

while counter <= 8:
    inputPrice = float(input(f"Digite o preço do produto {counter}: "))
    if(inputPrice > majorProduct):
        majorProduct = inputPrice
        
    if(inputPrice < minorProduct):
        minorProduct = inputPrice
    counter+=1
print(f"O maior preço: {majorProduct} \n O menor preço: {minorProduct}")

File: csharp/ex051.cs
// Faça um aplicativo que leia o preço de 8 produtos. No final, mostre na tela
// qual foi o maior e qual foi o menor preço digitados.


class Ex051
{
  public int counter = 1;
  public int soma = 0;

  public int major = 0;
  public int minor = 999;

  public void FirstCountdown()
  {
    while (counter <= 8)
    {
      Console.WriteLine($"Digite o preço do produto [{counter}]: ");
      float inputNumero = Convert.ToSingle(Console.ReadLine());

      if (inputNumero > major)
      {
        major = Convert.ToInt32(inputNumero);
      }
      
      if (inputNumero < minor)
      {
        minor = Convert.ToInt32(inputNumero);
      }
    }
    Console.Write($"Maiores : {major}");
    Console.Write($"Menores : {minor}");
  }
}

